tuf-cli/main.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/flynn/go-docopt"
+	tuf "github.com/flynn/go-tuf/client"
+)
+
+func main() {
+	log.SetFlags(0)
+
+	usage := `usage: tuf-cli [-h|--help] <command> [<args>...]
```

Comment:
Maybe call it `tuf-client`? Could put it in `cmd/tuf-client` and `tuf` in `cmd/tuf`.